flexget/plugins/input/ftp_list.py

```diff
@@ -143,8 +148,7 @@ def _handle_path(self, entries, ftp, baseurl, path='', mlst_supported=False, fil
                                   recursive, get_size, encoding)
 
             if not files_only or mlst.get('type') == 'file':
-                url = baseurl + path + '/' + p
-                url = url.replace(' ', '%20')
+                url = baseurl + urllib.quote_plus(path) + '/' + urllib.quote_plus(p)
```

Comment:
Ah yes, you're right. I guess the reason it worked was that I use the unquote_plus option in ftp_download. I'm actually a little confused as to what the correct behavior should be. i.e. Since my fix works with the ftp_list and ftp_download input+output combination I assumed I was good. However, if a different combination was used it might not work. Would the correct behavior be for me to add the path separator to the safe parameter in the call?